### DISCUSSION TITLE
[New pattern request] Clock widget to compliment the "calendar-month" component

### DISCUSSION THREAD

**Discussion by lclay2** [patternfly/patternfly#5487] [Feature requests]
**Requesting new feature**
- Requesting a clock widget to compliment the "calendar-month" component.
- The general use case is offering Date/Time settings during an installation. The date/time can be inferred from the system. The user will review the proposed current date and time, and accept it if it’s accurate or they will need to adjust it to arrive at the correct date/time. 
- To enable the user to scan the display to review the date and time the team is interested in using the “Calendar month
“ component (https://www.patternfly.org/v4/components/calendar-month) but we would like to pair that with a comparable Clock component.  
-- Someone on the team identified an open source, React clock component that has some nice features: https://projects.wojtekmaj.pl/react-clock/  
- Some general functional requirements are:
--Styled to compliment calendar component.
--Basic analog clock display, with hour and minute hands…
--Renders updates smoothly/quickly.
-- Accessible for visually impaired users. 
- The need for a clock widget is coming from the Anaconda installer project. There are some initial, conceptual design mockups, please refer to the attached screenshot.
<img width="632" alt="clockforcalendar" src="https://user-images.githubusercontent.com/10948231/200588495-3418f809-8cd0-4bdf-b2df-04e254be85e3.png">

**Comment by mcarrano** [comment]
Transferring for more discussion
